src/client/data/notificationMessages.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
should we remove this file?

Reply:
Lets leave it since we better have the importing from this file already implemented.